ggj25/scripts/bubball.gd

```diff
@@ -107,3 +101,10 @@ func find_ducks() -> Array[Duck]:
 		if n is Duck:
 			result.append(n as Duck)
 	return result
+
+
+func _on_hud_unpause_game() -> void:
```

Comment:
Does this also need to be added to dodge bubbal?

Reply:
yes